blogpost_14.html

```diff
@@ -270,11 +249,12 @@ <h3>Appearance Settings</h3>
                   <p>Dark themes now are identified and supported in a more reliable way. (Relevant e.g. for the Qogir and Orchis themes)</p>´
 
                 <h3>Display Settings</h3>
-                  <p>When several display profiles do match, the default display profile will be used now.</p>
-                  <p>Display profile names can now be edited.</p>
+                  <p>The default display profile has been added to the list of profiles that can match, and will be used if more than one matches now.</p>
+                  <p>Profile management has been extended to all profiles, not just those that match, and their names can now be edited.</p>
                   <p>Scaling now always is done for both dimensions and the preview shows the resulting size.</p>
-                  <p>It is now allowed to mirror outputs which are disabled.</p>
-                  <p>Finding a valid mode by refresh rate now works as expected.</p>
+                  <p>Mirror state management has been improved overall, in particular by removing the need to match refresh rates.</p>
+                  <p>The minimal dialog has been improved, in particular by allowing the display to be extended to the left and from top to bottom.</p>
+                  <p class="note">@Alex: Maybe a screenshot here?</p>
```

Comment:
Yea, fine for me to add a screenshot :+1:

Would be great if you could add one … though I as well can take care of it, if you prefer.

Reply:
Ah ok, I'll look into it, I wasn't sure if I could/should do it at my level (I rarely get my hands into html etc. ^^)